indicators/slurm.py

```diff
@@ -0,0 +1,251 @@
+"""Functions to assist with constructing slurm jobs"""
+
+import argparse
+import subprocess
+from config import *
+
+
+def make_sbatch_head(slurm_email, partition, conda_init_script, exclude_nodes, ncpus):
```

Comment:
```suggestion
def make_sbatch_head(partition, conda_init_script, exclude_nodes, ncpus):
```

I see the email option isn't used, and I have been doing away with it when I see it as well as I just don't find it very helpful. I would also recommend removing the exclude_nodes option, again don't think it's very useful - I remember putting it there (at least on some branch if it wasn't this) to get around one node that was causing trouble.